packages/testing/src/framework/test_fixtures/base.py

```diff
@@ -55,6 +59,20 @@ def __pydantic_init_subclass__(cls, **kwargs: Any) -> None:
         if cls.format_name:
             BaseFixture.formats[cls.format_name] = cls
 
+    @computed_field  # type: ignore[prop-decorator]
```

Comment:
I want to understand the goal here a bit more. I think I was originally a bit more uncertain about this, but the env var is set once at import time only right? So if we have this as cached _and_ as a computed field, it will still cache the very first time it's set, iiuc.

What if we made this a regular property and _didn't_ cache it? Something like:

```py
from lean_spec.subspecs.xmss import LEAN_ENV


class BaseFixture(CamelModel):
    lean_env: str = Field(default=LEAN_ENV)
    ...
```

I could very well be missing something. Lmk if that makes sense.